app/views/admin/settings/forms/_authentication.html.erb

```diff
@@ -14,6 +14,14 @@
         <h3 class="crayons-subtitle-3 mb-3">
           General settings
         </h3>
+        <div class="crayons-field">
```

Comment:
```suggestion
        <div class="crayons-field mb-6">
```

This class name will add a margin bottom to the group, which I think will solve the spacing issue you identified - this is the "after":


![field with additional spacing below](https://user-images.githubusercontent.com/20773163/142231778-76d3e3db-de2b-4f8b-8b9a-04816e093c1d.png)

Spacing-wise I thought this looked about right to make it look distinct from the next "Invite-only mode" setting - otherwise (to me at least) it looked like the next checkbox was somehow related to the text field before it